tests/models/decoder_only/language/test_jamba.py

```diff
@@ -275,6 +275,48 @@ def test_state_cleanup(
                     "could be related to finished_requests_ids")
 
 
+@pytest.mark.parametrize("model", MODELS)
+@pytest.mark.parametrize("dtype", ["float"])
+def test_multistep(
+    vllm_runner,
+    model: str,
+    dtype: str,
+    example_prompts,
+) -> None:
+    # This test is verifying that multistep works correctly
+    #on mamba-like models
+    try:
+        with vllm_runner(model, num_scheduler_steps=8,
+                         max_num_seqs=2) as vllm_model:
+            vllm_model.generate_greedy([example_prompts[0]] * 10, 1)
+    except ValueError:
```

Comment:
@mzusman where will this value error be raised?

Reply:
Removed this clause, thanks!